core/templates/pages/community-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -95,6 +103,34 @@ export class ContributionOpportunitiesBackendApiService {
     });
   }
 
+  private _fetchFeaturedTranslationLanguages(
+      successCallback: (
+        featuredTranslationLanguages:
+        ReadOnlyFeaturedTranslationLanguage[]
+      ) => void,
+      errorCallback: (reason?: any) => void
```

Comment:
The optional parameter in this callback might not be necessary here: https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html#optional-parameters-in-callbacks

Reply:
Done